implements

SrYthan opened the pull request.

ThexXTURBOXx commented:
I have improved this PR a bit. Is everything still working on your end with the new setup(s)?

SrYthan commented:
Olá, não tive tempo para testar ainda, assim que der eu retorno

SrYthan commented:
acho que para o industrialcraft só faltou o storage das maquinas, no mais tudo funcionando corretamente!

![image](https://github.com/user-attachments/assets/44ed26d5-2359-4f29-86af-38cc56ce71eb)
![image](https://github.com/user-attachments/assets/de93fbaf-81f6-4b0a-a45a-ce88f882e90a)

Commit:
Improve 1.5.2

- Add Total Solars support
- Add Twilight Forest patches
- Add Thaumcraft patches
- Add IC² patches
- Add new invertAdvTT setting
- Improve a few item display names
- Improve hard crashes and error messages
- Improve IC² support by adding lots of new infos
- Add AppEng monitor support

SrYthan commented:
adicionei mais um handler para obter o storage das maquinas, irei fazer um push, se houver algum erro corrija por favor 


![image](https://github.com/user-attachments/assets/230ab66a-3509-420c-83b5-a7276e80f7b8)

ThexXTURBOXx commented:
Thanks for the patch. I have squashed everything into a single commit, which is easier to deal with. Does everything still work on your end?

SrYthan commented:
irei testar tudo e retorno caso haja algo que ficou fora do esperado, muito obrigado por atualizar o mod e deixa-lo mais parecido com as novas versões!

ThexXTURBOXx commented:
@SrYthan Hey there, just checking if there has been any update on your end? :)

SrYthan commented:
olá, me desculpe a demora, estive bastante ocupado estes dias e não consegui testar ainda, hoje irei testar e retorno.

ThexXTURBOXx commented:
I merged it for now.
If there are any more problems, feel free to create another PR or report them via issues! :)
